lib/improver/tests/generate_ancillaries/test_topographicmasks.py

```diff
@@ -111,29 +108,13 @@ def test_landband_cube(self):
             self.orography, self.landmask, self.land_key,
             self.land_threshold)
         self.assertEqual(result.attributes['Topographical Type'], 'Land')
-        self.assertEqual(result.coord('topographic_bound_lower').points,
+        self.assertEqual(result.coord('topographic_zone').points,
+                         (self.land_threshold[1] - self.land_threshold[0]) / 2)
```

Comment:
I think this should be:
```python
(self.land_threshold[1] + self.land_threshold[0]) / 2
```
Although, it might be easier to replace this with:
```python
np.mean(self.land_threshold)
```

It is a fluke that in this instance: 
```python
self.land_threshold[1] - self.land_threshold[0] = self.land_threshold[1] + self.land_threshold[0]
```
It might be worth adding another test for a higher band e.g. [500 600] to make sure that you're catching this potential bug.